src/utils/cache.rs

```diff
@@ -164,7 +163,14 @@ pub trait CacheExt: Cache {
         cache_duration: usize,
         error_cache_duration: usize,
     ) -> ApiResult<String> {
-        self.request_cached_advanced(client, url, cache_duration, error_cache_duration, false, 0)
```

Comment:
What are the flows where this method is still used?

Should we completely remove this method to enforce a timeout everywhere?

Reply:
I would rather do this cleanup in the refactor task, to avoid double work.

Reply:
But where do we use the "default" timeout now?

Reply:
line 172